src/lndk_offers.rs

```diff
@@ -590,7 +590,7 @@ impl InvoicePayer for Client {
 
         let blinded_payment_paths = tonic_lnd::lnrpc::BlindedPaymentPath {
             blinded_path,
-            total_cltv_delta: u32::from(cltv_expiry_delta) + 120,
```

Comment:
What was the context for this extra delta before?

Reply:
IIRC in earlier versions of our fork of ldk-sample, payments wouldn't work without this addition. I think I got the exact number from the notes in Carla's PR here: https://github.com/lightningnetwork/lnd/pull/7267 But I think CLN doesn't like this additional delta & newer version of LDK don't need it. I'll add a comment to the commit for more clarification.